samples/sample_multi_transcode/src/sample_multi_transcode.cpp

```diff
@@ -584,6 +584,8 @@ mfxStatus Launcher::ProcessResult()
     for (mfxU32 i = 0; i < m_pThreadContextArray.size(); i++)
     {
         mfxStatus _sts = m_pThreadContextArray[i]->transcodingSts;
+        mfxF64 _workTime = m_pThreadContextArray[i]->working_time;
```

Comment:
Usage of variables starting with underscore is a bad practice. It is not recommended since there are some rules which underscore variables are reserved and used by system. See here https://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier

I'd suggest removing underscore from here

Reply:
Done. Also change "_sts" to "sts_"